src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -272,4 +273,20 @@ object HttpGoogleServicesDAO extends GoogleServicesDAO with FireCloudRequestBuil
     val pipeline: HttpRequest => Future[GooglePriceList] = sendReceive ~> decode(Gzip) ~> unmarshal[GooglePriceList]
     pipeline(Get(FireCloudConfig.GoogleCloud.priceListUrl))
   }
+
+  def status: Future[SubsystemStatus] = {
+    try {
+      val storage = new Storage.Builder(httpTransport, jsonFactory, getBucketServiceAccountCredential).setApplicationName("firecloud").build()
+      val bucketResponse = storage.buckets().list(FireCloudConfig.FireCloud.serviceProject).executeUsingHead()
```

Comment:
does this actually make a HEAD request, i.e. returns no body? That would be good - otherwise this is a heavyweight request to make for status.